examples/transfer-currency-fee.yaml

```diff
@@ -0,0 +1,76 @@
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - bob: https://bob.example/
+    alice: https://alice.example/
+    banka: https://banka.example/
+    bankb: https://bankb.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/transfer-currency-fee.yaml
+'@graph':
+
+  # transfer of currency from alice to bob
+
+  - '@id': banka:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    '@type': Process
+    skos:note: Facilitate moving currency between accounts
+    inScopeOf: https://banka.example/
+
+  - '@id': banka:0468e9b9-4d54-4eee-9d3f-bc76db11710a
+    '@type': EconomicEvent
+    action: transfer
+    transferOf:
+    - possession
+    - ownership
+    outputOf: banka:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    resourceClassifiedAs: wd:Q4917 # US Dollar
+    provider: https://alice.example
+    receiver: https://bob.example
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+    # alice and banka record the exact resource (account) on their copies of the transfer
+    resourceInventoriedAs: banka:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
```

Comment:
would it get incremented or decremented? if any than which event stays responsible for that increment or decrement

Reply:
I think you would have to tell by if it alice is the provider, then it is a decrement to alice's account.  I admit, that is a bit fishy.  And complicated by the relationship between alice and the bank.

Wim has it so there is transfer in the independent view, and give and take in the trading partners' views, and has them modeled as subtypes.  That seems sort of where we could go, but I didn't like the 3 things having different actions in our model.